web/webkit/src/main/scala/net/liftweb/http/LiftRules.scala

```diff
@@ -2295,3 +2301,80 @@ trait FormVendor {
   private object requestForms extends SessionVar[Map[String, List[FormBuilderLocator[_]]]](Map())
 }
 
+abstract class LiftRulesSettingException(settingName: String, stackTrace: LiftRules.StackTrace, message: String) extends Exception(message) {
```

Comment:
LOL, the same thoughts went through my mind when I tossed these in here with the rest. :)